src/java.base/share/classes/java/net/Inet4Address.java

```diff
@@ -134,6 +160,57 @@ class Inet4Address extends InetAddress {
         holder().originalHostName = hostName;
     }
 
+    /**
+     * Creates an {@code Inet4Address} based on the provided {@linkplain
+     * Inet4Address##format textual representations} of an IPv4 address.
+     * <p> If the provided IPv4 address literal cannot represent a {@linkplain
```

Comment:
"textual representations" -> "textual representation"

Reply:
Thanks Michael, fixed in af87fab27148daa982ab1997a4346e8c29be7604